src/main/java/io/jenkins/plugins/checks/github/GitHubChecksPublisher.java

```diff
@@ -55,17 +64,20 @@ public GitHubChecksPublisher(final GitHubChecksContext context, @Nullable final
     public void publish(final ChecksDetails details) {
         try {
             GitHubAppCredentials credentials = context.getCredentials();
-            GitHub gitHub = Connector.connect(StringUtils.defaultIfBlank(credentials.getApiUri(), GITHUB_URL),
+            GitHub gitHub = Connector.connect(StringUtils.defaultIfBlank(credentials.getApiUri(), gitHubUrl),
                     credentials);
-            GHCheckRunBuilder builder = createBuilder(gitHub, new GitHubChecksDetails(details));
-            builder.create();
+
+            GitHubChecksDetails gitHubDetails = new GitHubChecksDetails(details);
+            createBuilder(gitHub, gitHubDetails).create();
             if (listener != null) {
-                listener.getLogger().println("GitHub checks have been published.");
+                listener.getLogger().printf("GitHub check (name: %s, status: %s) has been published.",
+                        gitHubDetails.getName(), gitHubDetails.getStatus());
             }
         }
         catch (IllegalStateException | IOException e) {
             String message = "Failed Publishing GitHub checks: ";
-            LOGGER.log(Level.WARN, message, e);
+            LOGGER.log(Level.WARNING, message
+                    + ToStringBuilder.reflectionToString(details, new MultilineRecursiveToStringStyle()), e);
```

Comment:
Security problem exists (spogbug: CRLF_INJECTION_LOGS) for the reflection method, is it a concerns? Should we add `toString` method for the API models?

Reply:
Yes, adding a toString helps during debugging as well.

Reply:
ok, I'll add them in the api plugin.